modules/simple_forms_api/app/form_mappings/vba_20_10207.json.erb

```diff
@@ -22,6 +22,7 @@
   "form1[0].#subform[3].CurrentMailingAddress_NumberAndStreet[0]": "<%= form.data.dig('veteran_mailing_address', 'street') %>",
   "form1[0].#subform[3].CheckBox1[0]": "<%= nil %>",
   "form1[0].#subform[3].Email_Address[1]": "<%= nil %>",
+  "form1[0].#subform[3].InsuranceNumber[0]": "<%= nil %>",
```

Comment:
Is insurance number always supposed to be nil?

Reply:
Yep, per this issue, it'll never come through: https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/1142